server/index.ts

```diff
@@ -1,16 +1,38 @@
 import express from "express";
 import path from "path";
+import dotenv from "dotenv";
+import { connectDatabase, getUserCollection } from "../src/utils/database";
+dotenv.config();
 
 const port = process.env.PORT || 3001;
 const app = express();
 
+if (!process.env.MONGODB_URI) {
+  throw new Error("No MongoDB URI dotenv variable.");
+}
+
 app.get("/api/hello", (_request, response) => {
   response.json({ message: "Hello from server" });
 });
 
 // Serve production bundle
 app.use(express.static("dist"));
 
+// Connect to MongoDB (password is encoded in dotenv)
+connectDatabase(process.env.MONGODB_URI).then(() =>
+  app.listen(port, () => {
+    console.log(`Example app listening at http://localhost:${port}`);
+  })
+);
+
+// Display all users in MongoDB
+app.get("/api/users", async (_request, response) => {
+  const characterCollection = getUserCollection();
+  const characters = characterCollection.find();
+  const allCharacters = await characters.toArray();
+  response.send(allCharacters);
```

Comment:
I'd think about the naming here. Is it users or characters that the function getUserCollection gives you?